### ISSUE TITLE
Include NMVOC for BB4CMIP

### ISSUE THREAD

**Issue by jkikstra** [iiasa/emissions_harmonization_historical#14]
Currently, the download script and processing script for gfed-bb4cmip does not include any volatile organic compounds.

Assuming that we do want this, we might need to download all of them (and do something similar to the NMVOC conversion in https://github.com/iiasa/emissions_harmonization_historical/blob/main/notebooks/0102_GFED4-prepare.py)?

Will also be good to check this with Jonas and Margreet.

**Comment by chrisroadmap** [comment]
I think it would be the `NMVOC-bulk` variable we need?

**Comment by chrisroadmap** [comment]
as an aside I couldn't work out how to directly obtain this dataset using `esgpull`. I found it on ESGF

**Comment by znichollscr** [comment]
I think these are the two links we need (I have no idea what has happened with getting this published, @chrisroadmap can you please add your problems using ESGF search etc. here: https://github.com/PCMDI/input4MIPs_CVs/discussions/138):

- https://esgf-data2.llnl.gov/thredds/dodsC/user_pub_work/input4MIPs/CMIP6Plus/CMIP/DRES/DRES-CMIP-BB4CMIP7-1-0/atmos/mon/NMVOC_bulk/gn/v20241014/NMVOC_bulk_input4MIPs_emissions_CMIP_DRES-CMIP-BB4CMIP7-1-0_gn_175001-189912.nc
- https://esgf-data2.llnl.gov/thredds/dodsC/user_pub_work/input4MIPs/CMIP6Plus/CMIP/DRES/DRES-CMIP-BB4CMIP7-1-0/atmos/mon/NMVOC_bulk/gn/v20241014/NMVOC_bulk_input4MIPs_emissions_CMIP_DRES-CMIP-BB4CMIP7-1-0_gn_190001-202212.nc

**Comment by jkikstra** [comment]
> I think these are the two links we need (I have no idea what has happened with getting this published, @chrisroadmap can you please add your problems using ESGF search etc. here: [PCMDI/input4MIPs_CVs#138](https://github.com/PCMDI/input4MIPs_CVs/discussions/138)):
> 
>     * https://esgf-data2.llnl.gov/thredds/dodsC/user_pub_work/input4MIPs/CMIP6Plus/CMIP/DRES/DRES-CMIP-BB4CMIP7-1-0/atmos/mon/NMVOC_bulk/gn/v20241014/NMVOC_bulk_input4MIPs_emissions_CMIP_DRES-CMIP-BB4CMIP7-1-0_gn_175001-189912.nc
> 
>     * https://esgf-data2.llnl.gov/thredds/dodsC/user_pub_work/input4MIPs/CMIP6Plus/CMIP/DRES/DRES-CMIP-BB4CMIP7-1-0/atmos/mon/NMVOC_bulk/gn/v20241014/NMVOC_bulk_input4MIPs_emissions_CMIP_DRES-CMIP-BB4CMIP7-1-0_gn_190001-202212.nc

I get a loading problem for both these links at the moment.

**Comment by chrisroadmap** [comment]
> I get a loading problem for both these links at the moment.

yeah I don't think the links work directly, they didn't for me, but the files exist and are downloadable

**Comment by znichollscr** [comment]
> I get a loading problem for both these links at the moment

Urgh ESGF playing silly buggers.

Does this link load? https://aims2.llnl.gov/search?project=input4MIPs&activeFacets=%7B%22mip_era%22%3A%22CMIP6Plus%22%2C%22institution_id%22%3A%22DRES%22%7D. If you go to page 9, can you find it there?

**Comment by znichollscr** [comment]
This is what you're looking for

<img width="1048" alt="Screenshot 2024-10-29 at 4 03 25 pm" src="https://github.com/user-attachments/assets/3c8000f3-3798-4864-9060-d8ca99430362">

**Comment by jkikstra** [comment]
> > I get a loading problem for both these links at the moment
> 
> Urgh ESGF playing silly buggers.
> 
> Does this link load? https://aims2.llnl.gov/search?project=input4MIPs&activeFacets=%7B%22mip_era%22%3A%22CMIP6Plus%22%2C%22institution_id%22%3A%22DRES%22%7D. If you go to page 9, can you find it there?

Yep that works, can find the dataset - but after clicking the download button I get a new connection timed out error

**Comment by znichollscr** [comment]
Included somewhere along the line, hence closing
